packages/data-store/src/entities/contact/StudentEntity.ts

```diff
@@ -0,0 +1,51 @@
+import {BaseContactEntity} from "./BaseContactEntity";
+import {BeforeInsert, BeforeUpdate, ChildEntity, Column} from "typeorm";
+import {IsNotEmpty, Validate, ValidationError, validate} from "class-validator";
+import {IsNonEmptyStringConstraint} from "../validators";
+import {ValidationConstraint} from "../../types";
```

Comment:
Why are we adding a Student Entity? That is not at all what we want. We discussed to be able to extend individuals with additional properties. We are not creating a product around students, so we should be flexible in supporting new individual types, without having to provide all explicit entities for these types.

Reply:
All instances of student are killed now